src/index.html

```diff
@@ -151,17 +155,96 @@ <h2 class="modal__title">New post</h2>
         <img class="modal__image" src="#" alt="" />
         <p class="modal__caption"></p>
-      </div>
-    </div>
-    <script src="./scripts/index.js" defer></script>
-    <script src="./scripts/validation.js" defer></script>
+    <h3 class="modal__title">Edit profile avatar</h3>
+    <button type="button" class="modal__close-button">
+      <img
+        src="<%=require('./images/Close_icon.svg')%>"
+        alt="Close icon"
+        class="modal__close-icon"
+      />
+    </button>
+    <form name="edit-avatar" class="modal__form" novalidate>
+      <label for="profile-avatar-input" class="modal__label">
+        Profile Picture
+        <input
+          id="profile-avatar-input"
+          type="url"
+          class="modal__input"
+          name="avatar"
+          placeholder="Paste the link to an image"
+          minlength="2"
+          required
+        />
+        <span id="profile-avatar-input-error" class="modal__error"></span>
+      </label>
+      <button type="submit" class="modal__submit-button">Save</button>
+    </form>
+  </div>
+</div>
+<div id="preview-modal" class="modal">
+  <div class="modal__content">
+    <button class="modal__close-button" type="button">
+      <img src="./images/Close_icon.svg" alt="Close icon" class="modal__close-icon" />
+    </button>
+    <img class="modal__image" src="#" alt="Preview image" />
+    <p class="modal__caption"></p>
+  </div>
+</div>
+<div class="modal" id="avatar-modal-close">
+  <div class="modal__content modal__content_type_preview">
+    <button
+      class="modal__close-button modal__close-button_type_preview"
+      type="button"
+    ></button>
+    <img class="modal__image" src="#" alt="" />
+    <p class="modal__caption"></p>
+<h3 class="modal__title">Edit profile avatar</h3>
+<button type="button" class="modal__close-button">
+  <img
+    src="<%=require('./images/Close_icon.svg')%>"
+    alt="Close icon"
+    class="modal__close-icon"
+  />
+</button>
+<form name="edit-avatar" class="modal__form" novalidate>
+  <label for="profile-avatar-input" class="modal__label">
+    Profile Picture
+    <input
+      id="profile-avatar-input"
+      type="url"
+      class="modal__input"
+      name="avatar"
+      placeholder="Paste the link to an image"
+      minlength="2"
+      maxlength="40"
+      required
+    />
+    <span id="profile-avatar-input-error" class="modal__error"></span>
+  </label>
+  <button type="submit" class="modal__submit-button">Save</button>
+</form>
+</div>
+</div>
+
+<div class="modal" id="delete-modal">
+  <div class="modal__content">
+    <h2 class="modal__title">Are you sure you want to delete this image?</h2>
+    <button type="button" class="modal__close-button">
+      <img
+        src="<%=require('./images/Close_icon.svg')%>"
+        alt="Close icon"
+        class="modal__close-icon"
+      />
+    </button>
+    <form name="delete-card" class="modal__form_dlt">
+      <button type="submit" class="modal__button_dlt">Delete</button>
```

Comment:
`modal__button_dlt`

There is a BEM mistake

You can use `modal__button-dlt` as an element of `modal`